fastapi/openapi/utils.py

```diff
@@ -393,6 +393,7 @@ def get_openapi(
     terms_of_service: Optional[str] = None,
     contact: Optional[Dict[str, Union[str, Any]]] = None,
     license_info: Optional[Dict[str, Union[str, Any]]] = None,
+    prefix: Optional[str] = "",
```

Comment:
No nees optional. Just do this. 
```py
prefix: str=''
```

Reply:
Why don't you need "Optional"?

Reply:
Optional means it can be None. No need to be None.

Reply:
Done